performanceMetrics/src/App/App.js

```diff
@@ -183,11 +184,14 @@ const App = (props) => {
 
 	useEffect(() => {
 		if (listOfTestDates.length > 0) {
-			return setEndDate(convertBuildDateStringToMilis(listOfTestDates[listOfTestDates.length - 1]));
+			// include all entries from the latest date, regardless of their hour/minute/second
+			return setEndDate(convertBuildDateStringToMilis(listOfTestDates[listOfTestDates.length - 1]) + (1000 * 60 * 60 * 24));
```

Comment:
Actually I haven't tested this feature well. Under what conditions can I check?

After loading, the data is not immediately visible. I can see it if I change the period. Is it reproduced only on my environment?

Reply:
It was pretty tricky for me to test locally as well on the first time 😄 
Please make sure you have data in performanceMetrics/public (it must be a mirror of the sandstone folder from the server. it must include the versions folder, the develop folder and the 2 txt.files

To see today's test results
Please do the following
1. delete previous test results files
2. run the tests
3. copy all generated files to performanceMetrics/public/develop/results-YYYYMMDDHHmmss (create the folder first), were YYYYMMDD is today's date
4. Open frontend app. Today's date is the default end date and you should see results from today in the develop data   tab

List of test dates is generated on the folder names from developTestData.txt and  contains only the year-month-day info which generates a 00:00:00 time of that day, while the timestamp in the test results file is in milis. That's why I added an extra day in milis to avoid missing the results from the specified "End Date"